cudamapper/include/claragenomics/cudamapper/index.hpp

```diff
@@ -88,7 +88,7 @@ class Index
     /// \param hash_representations - if true, hash kmer representations
     /// \return instance of Index
     static std::unique_ptr<Index>
-    create_index(const io::FastaParser& parser,
```

Comment:
Same question about `const`

Reply:
As above, I don't think this is right or wrong, my preference is to not use `mutable`

Reply:
@tijyojwad fixed this, marking as resolved